_data/events.yml

```diff
@@ -196,6 +196,9 @@
 - name: "[Flink Forward (Apache Flink)](https://www.flink-forward.org/sf-2020)"
   status: cancelled, considering moving to virtual
 
+- name: "[Formula 1 Australian Grand Prix](https://grandprix.com.au/fan-zone/news/statement-regarding-formula-1-australian-grand-prix)"
```

Comment:
Would you mind adding the rest of the F1 races that have been cancelled/postponed (Bahrain, China, Vietnam IIRC)?